### DISCUSSION TITLE
Could not find Fan Controller (AMD Issue)

### DISCUSSION THREAD

**Discussion by hsellik** [devcompl/TTController#134] [Q&A] [answered]
Hi,

Before I start trying to make it work, does it help with the `Could not find Fan Controller, please check the Fan Controller is connected. (Error Code:H_0x0001)` issue that the original Thermaltake controller has with the AMD fans?

Tried to connect the controller to the back of the motherboard as some people have suggested, but it still does not work with the TT RGB Plus software. So I thought that maybe this software works for some odd reason?

Cheers

**Comment by MoshiMoshi0** [comment, marked as answer]
The issue with AMD was that TT RGB Plus could not read AMD cpu temperature. If your controller cant be found on any port then you probably have to RMA it.

**Reply by hsellik** [reply]
Hi, @MoshiMoshi0 

Thanks for the quick feedback. Today, I got a notification from the software saying that they fixed the AMD CPU temperature issue. I haven't yet tried it. 

Though, I suspect that reading my CPU temperature and finding my fan controller are NOT linked. I have not updated the software yet, but I can clearly see my CPU temperature on the TT RGB Plus software.

```
TT RGB Plus
Current software version:1.3.9
Latest software version:V1.4.0

1.Remove Music mode. 2.Fixed AMD CPU temperature reading error. 3.Fixed profile setting error.  4.Fixed IP address reading failure.  5.Support ARGENT Gaming products RGB SYNC.   6.Minor bug fixes and system performance.
```

**Reply by MoshiMoshi0** [reply]
Yes, they are not linked. All AMD issues I know of were a temperature sensor issue since they dont update their temperature sensor library often so newer AMD cpus didnt work.
The controller probably needs to be RMA'd like I said, but you can try checking debug report in TTController or updating TT RGB PLUS.
